src/minoltamn_int.cpp

```diff
@@ -1535,9 +1535,8 @@ static std::ostream& resolveLens0x34(std::ostream& os, const Value& value, const
     std::string model = getKeyString("Exif.Image.Model", metadata);
     std::string maxAperture = getKeyString("Exif.Photo.MaxApertureValue", metadata);
     long focalLength = getKeyLong("Exif.Photo.FocalLength", metadata);
-    std::string F2_8 = "760/256";
 
-    if (model == "SLT-A77V" && maxAperture == F2_8) {
+    if (model == "SLT-A77V" && maxAperture == "760/256") {
```

Comment:
Hum, this change is a bit controversial. We lose some contextual information that was present in the variable name. If you really want to remove the variable I would add a comment explaining that it corresponds to F2.8

Reply:
Fixed.